code/modules/antagonists/traitor/contractor/datums/objective_contract.dm

```diff
@@ -91,7 +91,7 @@
 			"Security Checkpoint",
 			"Technical Storage",
 			"Teleporter",
-			"Toxins Storage",
+			"Science Toxins Storage",
```

Comment:
Someone changed the name from `Toxins Storage` to `Science Toxins Storage` without updating it here. So I updated it here.